lib/python/grassdb/create.py

```diff
@@ -30,8 +30,8 @@ def create_mapset(database, location, mapset):
     # os.chmod(os.path.join(database,location,mapset,'WIND'), 0644)
 
 
-def get_default_mapset_name():
-    """Returns default name for mapset."""
+def get_current_user():
+    """Returns current user."""
```

Comment:
I think that "get_current_user" is a much more generic name and makes more sense even in create_mapset_interactively function where the default is set to get_current_user(). It means that immediately we know that default mapset name is the current user's name. 
By the way, this function is so far used only in guiutils when switching mapset and when creating a new mapset.

Reply:
I would let the name get_current_user() but maybe moved that function to grassdb/checks. I have noticed that in checks.py there are functions like get_mapset_owner, get_lockfile_if_present which have similar behavior = we try to GET something. And those functions do not perform any checks. Similarly like get_current_user(). It might be the reason why to set up different let's say get.py file.